src/elements/content-explorer/ItemGridThumbnail.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import getProp from 'lodash/get';
+import { getIcon } from '../common/item/iconCellRenderer';
+import './ItemGridThumbnail.scss';
+
+type Props = {
+    item: BoxItem,
+    token: Token,
```

Comment:
`Token` can be a string or a function, which the latter seems like it isn't handled below.